tasks/dojo.js

```diff
@@ -176,6 +176,54 @@ module.exports = function(grunt) {
       if(options.action){
        addParam('--action', options.action);
       }
+
+      if (options.layerOptimize != undefined){
```

Comment:
If only we had a way of adding messages to commits to tell people about these things.... ;)